app/controllers/admin/content/system-images/subtopics.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+export default Controller.extend({
+  actions: {
+    openModal(subtopic) {
+      if (subtopic.name === 'Avatar') {
```

Comment:
Don't set it like this. Do it directly on the template using the `array` helper 

Something like 
```hbs
aspectRatio=(if (eq subTopic.name 'avatar') (array 1 1))
```

No need to pass in 2:1 ... Let the default be 2:1